mpmp/src/cmds/Whisper.java

```diff
@@ -0,0 +1,43 @@
+package cmds;
+
+import java.util.Arrays;
+import main.Client;
+import main.Conn;
+
+/**
```

Comment:
Missing one-liner "C->S whisper packet". It should be C->S in your commit message, am I right?